src/lib/components/ui/Modal/Modal.jsx

```diff
@@ -30,20 +26,30 @@ class Modal extends React.Component {
     window.document.removeEventListener('keydown', this.pressEscapeHandler, false);
   }
 
-  setGradient() {
-    if (!this.state.isContentOverflowing) {
-      this.setState({ isContentOverflowing: true });
-    }
-  }
-
   pressEscapeHandler(e) {
-    if (e.keyCode === 27 && this.props.closeHandler) {
+    const { closeHandler } = this.props;
+
+    if (e.keyCode === 27 && closeHandler) {
       this.props.closeHandler();
```

Comment:
If you use object destructiong, replace `this.props.closeHandler();` with `closeHandler()` only

Reply:
Someone must have forgotten it, damn it.